src/commands/synthetics/api.ts

```diff
@@ -213,7 +214,18 @@ const createMobileVersion = (request: (args: AxiosRequestConfig) => AxiosPromise
 
 type RetryPolicy = (retries: number, error: AxiosError) => number | undefined
 
+const retryOnNodeErrors: RetryPolicy = (retries, error) => {
+  if (retries < MAX_RETRIES && isNodeError(error)) {
+    return LARGE_DELAY_BETWEEN_RETRIES
+  }
+}
+
 const retryOn5xxErrors: RetryPolicy = (retries, error) => {
+  const retryOnNodeErrorsDelay = retryOnNodeErrors(retries, error)
+  if (retryOnNodeErrorsDelay) {
+    return retryOnNodeErrorsDelay
+  }
```

Comment:
I wanted to keep the retry policy form, where `retryOnNodeErrors` could be used on its own if we wanted to.

```mermaid
flowchart BT
    a["retryOnNodeErrors"]
    b["retryOn5xxErrors"]
    c["retryOn5xxOr404Errors"]
    c --> b --> a
```

Reply:
But I understand realistically, it would not make sense on its own, as we would ignore the backend's response.

Around the naming, the "right" way would be to rename the policies to `retryOnNodeOr5xxErrors` and `retryOnNodeOr5xxOr404Errors`, but let's argue that retrying on Node errors should be intrinsic to any retry policy.